pjrt/pjrt.zig

```diff
@@ -140,13 +140,52 @@ pub const Api = struct {
         };
     }
 
+    pub inline fn stableHLOCurrentVersion(self: *const Api) ?[]u8 {
```

Comment:
this is weird

not sure I like this global variable set with a once. Maybe move this to the plugin load time ? so we can store it in the Context ?